certbot/tests/util_test.py

```diff
@@ -520,13 +521,13 @@ def test_systemd_os_release(self):
                 test_util.vector_path("os-release"))[0], 'systemdos')
             self.assertEqual(get_os_info(
                 test_util.vector_path("os-release"))[1], '42')
-            self.assertEqual(get_systemd_os_info("/dev/null"), ("", ""))
```

Comment:
Since we're testing the case when the file doesn't exist down below, I believe you can test the same behavior on Windows by using `os.devnull` here.

Reply:
Answer is 42. Your argument is invalid.

Done with os.devnull